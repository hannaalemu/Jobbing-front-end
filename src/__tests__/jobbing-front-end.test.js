import React from 'react';

import renderer from 'react-test-renderer';

import Enzyme, { shallow } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';

import App from '../app';

Enzyme.configure({ adapter: new Adapter() });

describe('< App /> ', () => {
  it('form is rendered at start', () => {
    const app = shallow(< App />);

    expect(app.find('span').exists()).toEqual(false);
  });

  it('matches snapshot', () => {
    const snapshot = renderer.create(< App />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
