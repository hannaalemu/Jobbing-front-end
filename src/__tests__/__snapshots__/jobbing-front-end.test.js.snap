// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< App />  matches snapshot 1`] = `
<h1>
  Hello World 
</h1>
`;
